src/main/java/handling/HTTP/requests/Sending/And/Processing/controller/ActivityTableController.java

```diff
@@ -36,50 +41,47 @@ public void createRequest() {
             }
         }
     }
+    public Void parseAndSaveInBD(String responseBody) {
 
-    public static Void parse(String responseBody) {
-
-        StringBuilder sb = new StringBuilder().append("[").append(responseBody).append("]");
-        JSONArray model = new JSONArray(String.valueOf(sb));
-
-        ActivityTableRrepositoryIml modelRrepositoryIml = new ActivityTableRrepositoryIml();
-        ActivityTableServise modelServise = new ActivityTableServise(modelRrepositoryIml);
+        model = new JSONArray("[" + responseBody + "]");
+        activityTableRepositoryIml = new ActivityTableRrepositoryIml();
```

Comment:
you should not create a repository and service here, you should inject them with the constructor.